utils/automatic-release/process.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+const spawn = require('cross-spawn');
+const fs = require('fs-extra');
+const path = require('path');
+const yaml = require('yamljs');
+const differ = require('jest-diff');
+const diffToSemver = require('./diffToSemver');
+const console = require('console');
+
+// run lernaUpdate and return a list of the packages that need to be updated
+const root = path.join(__dirname, '../../');
+const latestTemp = path.join(__dirname, './tempNpm');
+
+const checkUPdated = () => {
+  const cmd = spawn('npm', ['run', 'updated'], {
```

Comment:
I found you can do `lerna updated --json --loglevel=silent` for just getting this output in json. I think this could be a bit more robust?

Example:
```js
node node_modules/.bin/lerna updated --json --loglevel=silent
[
  {
    "name": "oc-template-react-compiler",
    "version": "5.0.0",
    "private": false
  },
  {
    "name": "react-app",
    "version": "1.0.19",
    "private": true
  }
]
```
In this way I think you can also easily skip the private ones here - which I think it would be nice?

Reply:
OMG - love you!